x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -66,6 +66,8 @@ export class CsvGenerator {
           index: indexPatternTitle,
           keep_alive: duration,
           ignore_unavailable: true,
+          // @ts-ignore
+          ignore_throttled: settings.includeFrozen ? false : undefined, // "true" will cause deprecation warnings logged in ES
```

Comment:
This is strange, but in my testing I found that this option doesn't seem to do anything when used with PIT. It seems like when PIT is used the search always includes frozen indices. 

This leads to the situation that when include frozen is false (default), discover won't show them, but csv will

Reply:
Hm, is there not a way to ignore frozen indices when using PIT? I don't see that mentioned in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html). Can we run this by the ES search team?

Reply:
I've reached out to the ES team, to get ideas why this happens.

Unfortunately, the change in the PR doesn't really seem to do anything, and has the side-effect of adding a `ts-ignore` line, since `ignore_throttled` isn't part of the `OpenPointInTimeRequest` interface. I believe we can probably drop awareness of the `includeFrozen` setting from this code, to resolve the bug and not add the `ts-ignore` line.

Reply:
Let's keep this here, but link to https://github.com/elastic/elasticsearch/pull/94377 after the `ts-ignore`, so anyone reading the code can understand why it is ignored. When the ES issue is resolved, the functionality should just work as the user intends, and at that point we might be able to just remove the `ts-ignore` as part of minor cleanup.

Reply:
I have merged https://github.com/elastic/elasticsearch/pull/94377. I think you can remove this workaround.

Reply:
Opened to clean up: https://github.com/elastic/kibana/pull/153359